### PR TITLE
Fix typos: not-build vs not-built

### PR DESCRIPTION
**Pull request by lifeofguenter**
I don't think we can change the config option, but also I don't think its too important. We can at least fix the help messages :)

wdyt @famod ?

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +0,0 @@
-* text=auto
```
Comment (KalleOlaviNiemitalo) [inline .gitattributes:1]: Why this change? Seems unrelated to typos in help messages

Comment (lifeofguenter) [reply .gitattributes:1]: Yah that's true.. I don't think that attribute works very reliably across platforms - or at least it does not seem to do anything, all files are still being pushed as CRLF and thus failing checkstyle. I played around with enforcing gitattributes, but its still not really reliable.

Instead, I think [`.editorconfig`](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/master/.editorconfig) is more reliable, which is also understood by GitHub and IDE's.

**Comment by famod** [comment]
Sorry, was busy elsewhere. LGTM!
